### DISCUSSION TITLE
ImageMagick Image Processing Design Error Code Execution Vulnerabilities - Zero Day

### DISCUSSION THREAD

**Discussion by priya86** [ImageMagick/ImageMagick#2870] [Development]
Hello,

We get the following security vulnerability when our team scanned the server. 
ImageMagick Image Processing Design Error Code Execution Vulnerabilities - Zero Day

Any idea how this can be fixed?

Appreciate all your help

**Comment by urban-warrior** [comment]
It could be a false positive.   We need more information before we can help.  What ImageMagick distribution were you downloading?  What is the link?  Which scanner are you using?   Currently, there are no known zero day vulnerabilities in ImageMagick.
